updatecli/updatecli.d/terraform.yml

```
@@ -0,0 +1,55 @@
+---
+name: Bump TERRAFORM version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  lastReleaseVersion:
+    kind: githubrelease
+    name: Get the latest TERRAFORM version
+    spec:
+      owner: "hashicorp"
+      repository: "terraform"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionfilter:
+        kind: semver
+    transformers:
+      - trimprefix: v
+
+conditions:
+  checkIfWindowsReleaseIsAvailable:
+    kind: shell
+    disablesourceinput: true
+    spec:
+      command: curl --connect-timeout 5 --location --head --fail --silent --show-error https://releases.hashicorp.com/terraform/{{ source `lastReleaseVersion` }}/terraform_{{ source `lastReleaseVersion` }}_windows_amd64.zip
+
+targets:
+  updateVersion:
+    name: "Update the TERRAFORM version in the provisioning/tools-versions.yml file"
+    sourceid: lastReleaseVersion
+    kind: yaml
+    spec:
+      file: "provisioning/tools-versions.yml"
+      key: "terraform_version"
+    scmid: default
+
+actions:
+  default:
+    kind: github/pullrequest
+    title: Bump TERRAFORM version to {{ source "lastReleaseVersion" }}
```

```suggestion
    title: Bump`terraform` version to {{ source "lastReleaseVersion" }}
```

Nit: indicates a CLI in the markdown based titles